using_images/xpaas_images/decision_server.adoc

```diff
@@ -0,0 +1,137 @@
++
+The results of a successful Maven build are installed into the local Maven repository, *_/home/jboss/.m2/repository/_*, along with all dependencies if the image is configured for offline-mode.
++
+Once the artifacts have been installed into the local Maven repository the resulting kjars are placed into *_EAP_HOME/standalone/deployments_* to deploy the rule. This includes all JAR, WAR, and EAR files from the directory within the source repository specified by `*$ARTIFACT_DIR*` environment variable. The default value of `*$ARTIFACT_DIR*` is the *_target_* directory.
+. Any JAR, WAR, and EAR in the *_deployments_* source repository directory are copied to the *_EAP_HOME/standalone/deployments_* directory.
+. All files in the *_configuration_* source repository directory are copied to *_EAP_HOME/standalone/configuration_*.
++
+[NOTE]
+If you want to use a custom JBoss EAP configuration file, it should be named *_standalone-openshift.xml_*.
+. All files in the *_modules_* source repository directory are copied to *_EAP_HOME/modules_*.
+
+=== Using a Modified Decision Server xPaaS Image
+
+An alternative method is to make changes to the image, and then use that modified image in OpenShift.  The templates currently provided, along with the interfaces they support, are listed below:
+
+.Provided Templates
+|=====================================
+| Template Name | Supported Interfaces
+| decisionserver62-basic-s2i.json | http-rest, jms-hornetq
+| decisionserver62-https-s2i.json | http-rest, https-rest, jms-hornetq
+| decisionserver62-amq-s2i.json | http-rest, https-rest, jms-activemq
+|=====================================
+
+You can run the Decision Server xPaaS image in Docker, make the required configuration changes using the JBoss EAP Management CLI (*_EAP_HOME/bin/jboss-cli.sh_*) included in the Decision Server xPaaS image, and then commit the changed container as a new image. You can then use that modified image in OpenShift.
+
+[IMPORTANT]
+It is recommended that you do not replace the OpenShift placeholders in the JBoss EAP xPaaS configuration file, as they are used to automatically configure services (such as messaging, datastores, HTTPS) during a container's deployment. These configuration values are intended to be set using environment variables.
+
+[NOTE]
+Ensure that you follow the   link:../../creating_images/guidelines.html[guidelines for creating images].
+
+=== Updating Rules 
+
+As each image is built from a snapshot of a specific Maven repository, whenever a new rule is added, or an existing rule modified, a new image must be created and deployed for the rule modifications to take effect.  
+
+== Troubleshooting
+
+In addition to viewing the OpenShift logs, you can troubleshoot a running Decision Image container by viewing the console logs.  These are outputted to the container's console, and are accessible with the following command:
```

Comment:
The OpenShift docs usually only use the term "console" in the context of "web console", so I think that's what Brice is taking it to mean here (e.g., "Events" tab in the webUI).